### PR TITLE
Set custom_container in build docs workflows

### PR DESCRIPTION
**Pull request by Wauplin**
**TL;DR:** this PR doesn't change anything but prevents future break.

Related to https://github.com/huggingface/doc-builder/pull/487. There is now a `custom_container` option when building docs in CI. When set to `""` (instead of `"huggingface/transformers-doc-builder"` by default), we don't run the CI inside a container, therefore saving ~2min of download time. The plan is to test disabling the transformers container on a few "big" repo and if everything works correctly, we will stop making it the default container. More details on https://github.com/huggingface/doc-builder/pull/487. This PR explicitly set the custom_container to transformers's one to avoid any breaks.

cc @mishig25

### CODE REVIEW

**Review by ArthurZucker — APPROVED** [review]
Thanks, lgtm

**Comment by Wauplin** [comment]
Thanks for reviewing :)
